analyzers/rolaguard_bruteforce_analyzer/BruteforcerMain.py

```diff
@@ -42,20 +42,21 @@ def process_packet(packet, policy):
 
             if len(pot_app_keys) > 0:
                 
-                keys_to_test=list()
-                
-                for pot_app_key in pot_app_keys:
-                    keys_to_test.append(bytes(pot_app_key.app_key_hex.rstrip().upper(), encoding='utf-8')) 
-
+                keys_to_test=[bytes(pk.app_key_hex.rstrip().upper(), encoding='utf-8') for pk in pot_app_keys] 
```

Comment:
The keys_to_test var isn't initialized? Although a one liner command seems nice, bear in mind that sometimes is not that easy to read

Reply:
It is a list comprehension, It is defined and initialized in the same line. I used it because It is more efficient than using append in a for loop and is considered more "pythonic" and declarative.